index.fr.md

```diff
@@ -0,0 +1,111 @@
+{:/}
+
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenu de la page" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Plus que "Mobile" {#intro}
+
+"L'accessibilité mobile" fait référence au travail visant à rendre les sites web et applications plus accessibles aux personnes souffrant de handicaps lorsqu'ils utilisent des téléphones mobiles et autres appareils. Le travail de la WAI dans ce domaine traite des problèmes d'accessibilité pour les personnes utilisant une large gamme d'appareils pour interagir avec le Web incluant :
+
+-   téléphones et tablettes
+-   télévisions numériques
+-   appareils portables tels que les montres connectées
+-   appareils dans les tableaux de bord de voitures ou dossiers de sièges d'avion
+-   appareils dans l'électroménager
+-   autres "internet des objets"
+
+Cela couvre une grande étendue de problèmes concernant :
+
+-   écrans tactiles
+-   écrans de petite taille
+-   modes de saisie variés, incluant la voix et la technologie "3D Touch" facilitée par
+    des capteurs de pression
+-   appareils utilisés avec des paramètres différents comme les écrans lisibles en plein soleil (transflectifs)
+-   et bien d'autres
+
+## W3C WAI traite l'accessibilité mobile {#covered}
+
+**Les standards d'accessibilité formulés par la WAI traite de l'accessibilité mobile** :
+
+-   **Les Règles pour l'accessibilité des contenus Web** ([WCAG](/standards-guidelines/wcag/)) traitent des pages et applications web, incluant le contenu à destination des appareils mobiles.
+    -   Pour apprendre comment WCAG 2.0 s'applique au contenu web pour mobile, aux applications web pour mobile, aux applications natives et aux applications hybrides utilisant des composantes web, voir [Accessibilité Mobile : Comment les WCAG 2.0 et autres règlements de la WAI du W3C s'appliquent au Mobile](http://www.w3.org/TR/mobile-accessibility-mapping/).
+    -   Une ressource plus générale qui inclut également les applications mobiles est : [WCAG2ICT : Appliquer les WCAG 2.0 à l'information non web et aux technologies de communication](http://www.w3.org/TR/wcag2ict/).
+    -   WCAG 2.1, les règles publiées en Juin 2018 contiennent de nouvelles exigences ("critères de succès") traitant de l'accessibilité mobile.  Elles sont présentées dans : [[Nouveautés dans les WCAG 2.1]](/standards-guidelines/wcag/new-in-21/).
+-   **Les Règles pour l'accessibilité des agents utilisateurs** ([UAAG](/standards-guidelines/uaag/)) traitent des navigateurs web ainsi que d'autres agents utilisateurs, y compris les navigateurs sur appareils mobiles.
+    -   Pour consulter des exemples sur les bénéfices accordés aux personnes souffrant de handicaps lors de la navigation web sur un appareil mobile, voir les [Exemples d'accessibilité Mobile dans le contexte des UAAG](http://www.w3.org/TR/IMPLEMENTING-UAAG20/mobile).
```

Comment:
- I suggest "personnes en situation de handicap" rather than "personnes souffrant de handicaps"
- The meaning of the translated version is a bit different from the original one. 
- Suggestion "Pour des exemples sur la manière dont des navigateurs web respectant les UAAG bénéficient aux personnes en situation de handicap qui utilisent le Web sur des appareils mobiles, consultez"
- Typo : "**W**eb"

Reply:
much better!